irohad/main/application.hpp

```diff
@@ -80,6 +80,8 @@ class Irohad {
    * @param pg_conn - initialization string for postgre
    * @param listen_ip - ip address for opening ports (internal & torii)
    * @param torii_port - port for torii binding
+   * @param torii_tls_port - port for torii TLS binding
+   * @param torii_tls_keypair - path to TLS keypair (.crt, .key) for torii
```

Comment:
Please add a remark about the path format. Should it include file extension or not?

Reply:
Fixed in latest version